src/languages/english.properties

```diff
@@ -27,8 +27,8 @@ ERROR_USER_NO_ACCESS_VARIABLE_DIRECT=The user doesn't have direct access to that
 
 VARIABLE_REMOVED_FROM_USER=Variable %s removed from the user %s.
 VARIABLE_REMOVED_FROM_GROUP=Variable %s removed from the group %s.
-VARIABLE_ADDED_TO_USER=Variable %s:'%s' added to the user %s.
-VARIABLE_ADDED_TO_GROUP=Variable %s:'%s' added to the group %s.
+VARIABLE_ADDED_TO_USER=Variable %s: '%s' added to the user %s.
+VARIABLE_ADDED_TO_GROUP=Variable %s: '%s' added to the group %s.

```

Comment:
These are intentional formatting spaces.

Reply:
What do you mean? It looks nicer to have a space after `:`

Reply:
Yes, its clearer in the final message